go/mcap/indexed_message_iterator.go

```diff
@@ -84,6 +99,7 @@ func (it *indexedMessageIterator) parseSummarySection() error {
 	}
 	defer lexer.Close()
 
+lexerloop:
```

Comment:
this is a small nitpick, but I'd consider calling this something like "Top". I recall using that convention elsewhere in Foxglove code but I don't have anything to point at - it's probably all in the DP. IMO "break Top" is a clearer expression to the reader than "lexerloop", since "Top" immediately conveys "topmost loop level over this one" where "lexerloop" would make me search for that label.

Reply:
yeah, i'll likely break this out to its own function instead of continuing with labelled breaks